index.js

```diff
@@ -19,9 +19,32 @@ mongoose
     // Before adding any recipes to the database, let's remove all existing ones
     return Recipe.deleteMany()
   })
-  .then(() => {
+  .then(() => Recipe.syncIndexes())
+  .then(() => {Recipe.create([{
+    title: 'Churrata',
+    level: 'Easy Peasy',
+    ingredients: [
+      '500grm churros',
+      '4 huevos',
+      'patata',
+      'cebolla'],
+    cuisine: 'Spanish',
+    dishType: 'breakfast',
+    image: 'https://images.media-allrecipes.com/userphotos/720x405/815964.jpg',
+    duration: 40,
+    creator: 'Jesús Sánchez Diezma',
+  }])
     // Run your code here, after you have insured that the connection was made
   })
+  .then((newRecipe => console.log (newRecipe)))
+  .then(()=> Recipe.create(data))
+  .then((newRecipe) => newRecipe.forEach(elm => console.log(elm.title)))
+  .then(()=> Recipe.findOneAndUpdate({duration: 220}, {duration:100}))
+  .then(()=>(console.log('Recetita creada')))
```

Comment:
- Cuando haces el `console.log` no te hace falta ponerlo entre paréntesis
```suggestion
.then(()=> console.log('Recetita creada'))
```